src/main/java/uk/gov/hmcts/reform/iahearingsapi/infrastructure/controllers/WelcomeController.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.iahearingsapi.infrastructure.controllers;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.springframework.http.CacheControl;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Bail service")
```

Comment:
All the Bail case API text can be changed IA Hearings API. Some on other classes too.

Reply:
Pushing a commit to update all of these references